src/components/utils/field-constants.js

```diff
@@ -227,3 +227,11 @@ export const VARIABLE_SHUNT_COMPENSATORS = 'variableShuntCompensators';
 export const PRIORITY = 'priority';
 export const FILTER_ID = 'filterId';
 export const FILTER_NAME = 'filterName';
+
+// HVDC deletion
+export const DELETION_SPECIFIC_DATA = 'specificData';
+export const DELETION_SPECIFIC_TYPE = 'specificType';
+export const HVDC_LINE_LCC_DELETION_SPECIFIC_TYPE = 'HVDC_LINE_WITH_LCC';
```

Comment:
Move to `equipment-types.js `file ?

Reply:
It is not a real type. It's used as a specific "sub-type" in deletion modification.
let's keep it here ?

Reply:
yes i think